### DISCUSSION TITLE
请问通过邮箱授权就可以找回激活码吗

### DISCUSSION THREAD

**Discussion by luyuan59** [getlantern/lantern-forum-cn#396] [General]
<img width="577" alt="截屏2024-04-06 01 24 30" src="https://github.com/getlantern/lantern-forum-cn/assets/3895510/ca5e7a81-5da5-4381-8637-3c4e47579d38">

以上是我的付款凭证，我的邮箱是「left199059@gmail.com」,请给我发送激活码。
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@luyuan59 是的。您可以尝试授权一下，看看是否可以使用它登录？

**Reply by luyuan59** [reply]
授权了，没有用，收不到邮件。要不然给我发送激活码，要不然给我退款吧。

**Reply by luyuan59** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@luyuan59 
嗯嗯. 不好意思让您久等了. 好的. 我帮您对接负责人帮您拿激活码. 要等星期二才能答复您了. 团队星期一深夜才回上线. 当我拿到激活码马上回复您!
